pkgs/top-level/all-packages.nix

```diff
@@ -12077,7 +12077,9 @@ with pkgs;
 
   sigil = libsForQt5.callPackage ../applications/editors/sigil { };
 
-  signalbackup-tools = darwin.apple_sdk_11_0.callPackage ../applications/networking/instant-messengers/signalbackup-tools { };
+  signalbackup-tools = callPackage ../applications/networking/instant-messengers/signalbackup-tools {
+    stdenv = if (stdenv.isDarwin && stdenv.isx86_64) then darwin.apple_sdk_11_0.clang14Stdenv else stdenv;
```

Comment:
added a additional condition on x86_64 because it builds on aarch64-darwin without the clang14Stdenv

Reply:
I don't think we should use a different version of Clang on different Darwin architectures. Clang 14 was being using since it was required for Darwin builds. If it isn't required on `aarch64-darwin` is shouldn't be required on `x86_64-darwin`.

Also, I don't love having this logic in `all-packages.nix`. I think it makes the package much harder to reason about.